### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by shadowcodex**
Fixes: Fixed as many Node.js references as I felt comfortable doing. 
Needed: Once new support links have been established then these links need to be updated. Examples include the freenode IRC channels, etc...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 ## Issue Contributions
 
 When opening issues or commenting on existing issues, please make sure
-discussions are related to concrete technical issues with Node.js.
+discussions are related to concrete technical issues with Ayo.js.
 
-* For general help using Node.js, please file an issue at the
-[Node.js help repository](https://github.com/nodejs/help/issues).
+* For general help using Ayo.js, please file an issue at the
+[ayo.js help repository](https://github.com/nodejs/help/issues).
```
Comment (ghost) [inline CONTRIBUTING.md]: should we leave this at `node.js help repository`? otherwise it's a misguided link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -107,9 +107,9 @@ of the log. Use the `Fixes:` prefix and the full issue URL. For other references
 use `Refs:`.
 
    Examples:
-   - `Fixes: https://github.com/nodejs/node/issues/1337`
+   - `Fixes: https://github.com/ayojs/ayo/issues/1337`
```
Comment (ghost) [inline CONTRIBUTING.md:110]: these would be dead links

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -308,14 +308,14 @@ weekend) from when it is submitted, even after it gets approved and
...
-merged in. All these precautions are important because Node.js is
+merged in. All these precautions are important because Ayo.js is
 widely used, so don't be discouraged!
 
 ### Check Out the Collaborator's Guide
 
 If you want to know more about the code review and the landing process,
 you can take a look at the
-[collaborator's guide](https://github.com/nodejs/node/blob/master/COLLABORATOR_GUIDE.md).
+[collaborator's guide](https://github.com/ayojs/ayo/blob/master/COLLABORATOR_GUIDE.md).
```
Comment (ghost) [inline CONTRIBUTING.md]: just to future-proof, could you make this a relative link?

**Review by ghost — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (addaleax) [reply CONTRIBUTING.md:110]: that’s okay, it’s fictional anyway

**Review by addaleax — APPROVED** [review]

**Comment by addaleax** [comment]
Looks good, but we should either set up a help repo of our own or just leave the link like @pup said

**Comment by ghost** [comment]
i think while our technical features largely overlap with node, we don't really need a dedicated help repo yet

**Comment by addaleax** [comment]
We don’t *need* one, but maybe we should see if people are *willing* to help in that fashion, at least at some point

File: CONTRIBUTING.md (hunk 2 above)
Comment (ghost) [reply CONTRIBUTING.md:110]: oh haha

File: CONTRIBUTING.md (hunk 1 above)
Comment (Kingles) [reply CONTRIBUTING.md]: it seems a little confusing to continue linking back to node's help repo.. can we make the new repo now or is something preventing that?

**Comment by shadowcodex** [comment]
Agreed, only do it if you have the contributors for it. I wouldn't want ya'll to pull focus away from the core repo work that needs to be done.

File: CONTRIBUTING.md (hunk 1 above)
Comment (varjmes) [reply CONTRIBUTING.md]: @Kingles we're doing small tasks and iterations right now to lay ground work. We're not ready to do the help repo just yet.

**Comment by varjmes** [comment]
> We don’t need one, but maybe we should see if people are willing to help in that fashion, at least at some point

Agree. But let's wait for a little bit until we've stabilised? And by wait, I mean like, let's think about this after the weekend or something and work out if we have the contributors.

**Comment by shadowcodex** [comment]
We could just point to the discord for now?

**Comment by Fishrock123** [comment]
We'd want a secondary discord channel in that case - easy enough though.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Qard) [reply CONTRIBUTING.md]: We could just point it at this repo for now and change it to point at another repo if we make another at some point in the future.

**Comment by addaleax** [comment]
Can we merge this? It doesn't seem meaningful to me, at least at this point, to hold it up over the (lack of a) help channel.

**Comment by Qard** [comment]
see: https://github.com/ayojs/ayo/pull/23#discussion_r135426033

**Comment by shadowcodex** [comment]
I made some of the changes. Made one link relative and the other link to just point back to this repo for now.

**Review by Fishrock123 — APPROVED** [review]
will merge by end-of-day (but feel free to merge sooner)

**Comment by Fishrock123** [comment]
@pup Mind dismissing your review or re-reviewing? It seems your feedback has been addressed?

**Review by Qard — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
whoops sorry, i'll land this now

**Comment by ghost** [comment]
landed in 5419aa5!

**Pull request closed without merging**